internal/analytics/env.go

```diff
@@ -83,3 +90,57 @@ func IsFaaS() bool {
 
 	return false
 }
+
+// binarySource identifies where the binary came from
+func binarySource() string {
+	switch {
+	case isHomebrew():
+		return "homebrew"
+	case isContainer():
+		return "container"
+	}
+	return "unknown"
+}
+
+// isHomebrew returns true if CloudQuery is running under Homebrew
+func isHomebrew() bool {
+	// Executable returns the path name for the executable that started the current process.
+	executable, err := os.Executable()
+	if err != nil {
+		return false
+	}
+	// Get the homebrew prefix
+	var out []byte
+	if out, err = exec.Command("brew", "--prefix").Output(); err != nil {
```

Comment:
Can we remove this at all for now? not a fan of executing on every command a new process.